src/main/java/io/jenkins/plugins/aws/global_configuration/AwsManagementLink.java

```diff
@@ -74,7 +77,8 @@ public synchronized void doConfigure(StaplerRequest req, StaplerResponse rsp) th
 
     public boolean configure(StaplerRequest req, JSONObject json) throws Descriptor.FormException {
         boolean result = true;
-        for(Descriptor<?> d : Functions.getSortedDescriptorsForGlobalConfig(FILTER)){
+        for(Descriptor<?> d : Functions.getSortedDescriptorsForGlobalConfigByDescriptor(descriptor ->
+                descriptor.getCategory() instanceof AwsGlobalConfigurationCategory)){
```

Comment:
or for clarity
```suggestion
        for(Descriptor<?> d : Functions.getSortedDescriptorsForGlobalConfigByDescriptor(descriptor ->
                FILTER.apply(descriptor.getCategory()))){
```

Reply:
> clarity

In who's eyes? 😄 

While writing this line I spent more time than I care to admit trying to figure out what `FILTER` is, and what I ended up writing made sense to me.

Reply:
The point of the constant was to share this logic with the view: https://github.com/jenkinsci/aws-global-configuration-plugin/blob/79f183811cfb8f880285fab58ed3fdc0fd0d2afb/src/main/resources/io/jenkins/plugins/aws/global_configuration/AwsManagementLink/index.groovy#L28

Reply:
Would probably be better to remove `FILTER` and instead expose a Java method wrapping `getSortedDescriptorsForGlobalConfigByDescriptor`.

Reply:
Gotcha! Somehow I didn't notice:
https://github.com/jenkinsci/aws-global-configuration-plugin/blob/041a055a44d4ddf3ae77f29d390567c31cb11a4f/src/main/java/io/jenkins/plugins/aws/global_configuration/AwsManagementLink.java#L96-L100